tools/ingest-converter/src/test/java/org/logstash/ingest/DateTest.java

```diff
@@ -0,0 +1,32 @@
+package org.logstash.ingest;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public final class DateTest extends IngestTest {
+
+    @Rule
+    public final TemporaryFolder temp = new TemporaryFolder();
```

Comment:
@suyograo I assume you ran into `IllegalAccessException` here and are therefore asking?